asset/src/_kafka_clients/consumer-client.ts

```diff
@@ -33,15 +37,51 @@ export default class ConsumerClient extends BaseClient<kafka.KafkaConsumer> {
         started: {},
         ended: {},
     };
-
+    private encoding: EncodingConfig = {};
     /** last known assignments */
     protected _assignments: TopicPartition[] = [];
 
     private _pendingOffsets: CountPerPartition = {};
     private _rebalanceTimeout: NodeJS.Timer|undefined;
+    private rollback_on_failure = false;
```

Comment:
We shouldn't be mixing casing like this, if the other class properties use camelCase we should keep the camelCase schema. This applies to `use_commit_sync` too.